CHANGELOG.md

```diff
@@ -1,216 +1,68 @@
 
 ## Breaking Changes
 
-| Description                                                                                                                                                                                                                          | PR       |
-|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
-| <!--PWA-1588-->Removed `Enzyme` and `@wojtekmaj/enzyme-adapter-react-17` packages. Update any tests that use the `Enzyme` or `@wojtekmaj/enzyme-adapter-react-17` packages.                                                          | [3393][] |
-| <!--PWA-1704-->Changed the UPWARD configuration to prevent a race condition that could prevent `yarn build` command from emmitting images. Overriden static asset configurations now only take the overriden value, instead of both. | [3410][] |
-| <!--PWA-1704-->Hundreds of CSS files have been renamed and their components updated to point to the new filenames.                                                                                                                   | [3414][] |
-| <!--PWA-1908-->Page loading indicator component refactored/moved to LoadingIndicator.                                                                                                                                                | [3308][] |
-| <!--PWA-1908-->`makeRoutes` has changed to also export a list of available routes.                                                                                                                                                   | [3308][] |
-| <!--PWA-1908-->Any use of `getUserConfirmation` external of Venia must be updated as we are now using this functionality.                                                                                                            | [3308][] |
-| <!--PWA-1204-->Response from `useMagentoRoute` has changed for loading state. Wrappers should take account of this change.                                                                                                           | [3353][] |
-| <!--PWA-1878-->Added exports to index.js files for Shimmers.                                                                                                                                                                         | [3284][] |
-| <!--PWA-1878-->Changed content of `categoryContent` root component so targetables may change.                                                                                                                                        | [3284][] |
+-  [3447][] — The fix for the cart rendering issue requires you to install our new meta-package. The meta-package adds a new GraphQL endpoint used to fix the cart rendering issue. This endpoint is not in then Commerce core.
```

Comment:
Perhaps our testing didn't catch this because you'd need to hit the very specific edge-case condition for it to fail?

Seems weird though, if the breakage is [because of the new errors fields](https://github.com/magento/pwa-studio/pull/3447/files#diff-9f77de1c810c09a21c2f550b8e4996ee08bbaa9e0de28e1f9e9f385c29595ea4), I would expect that to break even when there were no errors present.

Do we have to revisit the `13.0.0` vs `12.1.0` conversation if this is a breaking change?

cc @JustinConabreeAbsolunet @dpatil-magento

Reply:
> The meta-package adds a new GraphQL endpoint

Also, is this correct? More accurately it exposes some new fields on the existing `cart` query, right?

Reply:
@supernova-at I think the backend instances have been updated to have the new meta package. It's querying for the new `errors` field and isn't crashing, which would happen otherwise. @dpatil-magento might know more!

Personally I would lean more towards marking it as a breaking change (since it requires installing something on the backend in order for the application to work). But at the same time, would that mean any changes/additions made in the future to the metapackage be "breaking" since they need to update the backend module as well?
I think we can get away with it still being a `minor` so long as we very clearly mark external dependencies in the release notes going forward (Magento version, UPWARD PHP version, new metapackage version, etc)


> Also, is this correct? More accurately it exposes some new fields on the existing cart query, right?

Yeah exactly. Same existing endpoints, just a new field available on the CartItemInterface (anywhere cart items are used, this new field can be fetched)

Reply:
Thanks for Clarifying, @JustinConabreeAbsolunet. I'll fix the new endpoint statement and ponder the breaking changes wording(s).

Reply:
So for this release, we installed the metapackage on _our_ backend instances, but if anyone is accessing a different backend instance, it would break unless they install the metapackage themselves?

Reply:
Just committed updates to describe breaking changes.